conan/cli/commands/download.py

```diff
@@ -17,27 +21,45 @@ def download(conan_api: ConanAPI, parser, *args):
     queries over the package binaries.
     """
 
-    parser.add_argument('reference', help="Recipe reference or package reference, can contain * as "
-                                          "wildcard at any reference field. If revision is not "
-                                          "specified, it is assumed latest one.")
+    parser.add_argument('pattern', nargs="?",
+                        help="A pattern in the form 'pkg/version#revision:package_id#revision', "
+                             "e.g: zlib/1.2.13:* means all binaries for zlib/1.2.13. "
+                             "If revision is not specified, it is assumed latest one.")
     parser.add_argument("--only-recipe", action='store_true', default=False,
                         help='Download only the recipe/s, not the binary packages.')
     parser.add_argument('-p', '--package-query', default=None, action=OnceArgument,
                         help="Only download packages matching a specific query. e.g: os=Windows AND "
                              "(arch=x86 OR compiler=gcc)")
     parser.add_argument("-r", "--remote", action=OnceArgument, required=True,
                         help='Download from this specific remote')
+    parser.add_argument("-l", "--list", help="Package list file")
```

Comment:
Perhaps the `--graph-list` name fits better? I thought of this name like the param from `conan list` is  --graph`. Even a better help message could be enough... Wdyt?

Reply:
It is not a graph list, it is a package list, the result of the ``conan list`` command.

Reply:
Sure! That was I meant. I think it could be a bit confusing to say only `
Package list file` instead of:

```suggestion
    parser.add_argument("-l", "--list", help="Package list file. See more information in [Docs_url]/commands/list#format-json")
```
Or something like that. At least, pointing out some kind of documentation about that file.

Reply:
Too much in my opinion. Why this and not others? Following this, we should be adding links to the docs to many of the arguments of many commands, and note that links can read badly in the CLI help output.